packages/graphql_flutter/pubspec.yaml

```diff
@@ -8,7 +8,8 @@ authors:
 - Michael Joseph Rosenthal <rosenthalm93@gmail.com>
 homepage: https://github.com/zino-app/graphql-flutter/tree/master/packages/graphql_flutter
 dependencies:
-  graphql: ^3.0.1-beta.1
+  graphql:
+    path: ..\graphql
```

Comment:
Do not commit the path for graphql, this is for development.

Reply:
Ahh whoops. I'll fix that

Reply:
Just pushed a fix